lib/omniauth/strategies/census.rb

```diff
@@ -4,11 +4,22 @@ module OmniAuth
   module Strategies
     class Census < OmniAuth::Strategies::OAuth2
       include OmniAuth::Strategy
+
+      def self.provider_endpoint
+        
+        if ENV['CENSUS_ENV'] == 'production' || ENV['RACK_ENV'] == 'production'
+          return "https://turing-census.herokuapp.com"
+        else
+          return "https://census-app-staging.herokuapp.com"
+        end
```

Comment:
@neight-allen Would we be able to put an elsif in here that checks if one of the environments is development to return localhost:3001 where someone is running a local census development on port 3001?  Not knowing enough about how these two environments are set, is there one that wouldn't automatically be set to development when working in development? We wouldn't want all work in development to rely on a locally running version of Census.  Does this question make sense?